README.md

```diff
@@ -26,6 +26,17 @@ Further reading:
 uVisor in the press:
 - [Q&A with ARM](http://eecatalog.com/IoT/2015/08/18/qa-with-arm-securing-the-iot-using-arm-cortex-processors-and-a-growing-mbed-platform-suite/): Securing the IoT using ARM Cortex Processors, and a growing mbed platform suite
 
+### Warning: Enabling Debug Features on uVisor
+Please note that currently the latest version of uVisor (the `HEAD` of the `uvisor` repository) does not support the following features:
+* LED blinking patterns
+* Memory mapped debugging
+
+This means that the support for hardware-specific debugging is temporarily unavailable. If you want to revert back to a version of uVisor that still supports those features, please checkout the `uvisor_with_memory_map_debug` branch:
+```bash
+git clone --recursive -b uvisor_with_memory_map_debug git@github.com:ARMmbed/uvisor.git
+```
```

Comment:
I don't like recommending to clone an entirely new copy of the repo just for a new branch, but it's probably the easiest way to make sure you are using the right branches (at the expense of some disk space and inability to, for instance, cherry-pick between branches).